native/src/modules/i18n/components/I18nProvider.js

```diff
@@ -59,7 +59,9 @@ export default ({ children }: PropsType) => {
         debug: buildConfig().featureFlags.developerFriendly
       })
 
-      await setContentLanguage(i18nextInstance.language).catch(e => {
+      // A language mentioned in the supportedLanguages array of the config.js in the translations package
+      const matchedLanguage = i18nextInstance.languages[0]
+      await setContentLanguage(matchedLanguage).catch(e => {
```

Comment:
I don't understand this change. Could you explain why it was necessary?

Reply:
So the thing is chinese is our first language using a language tag instead of a code. That means we do not want to throw away everything except the code when detecting languages from browsers for example.

This means it is now possible to set de-DE as language. I use react-i18next to match de-DE automatically to a language we support (based on the config.js). This allows us to match 'zh-hans' to 'zh-CN' for example.